### PR TITLE
Yrsa 2.000

### PR DESCRIPTION
**Pull request by yanone**
The VFs show a vertical metrics failure. They are created from the same source as Rasa, just with a smaller character set, so personally I think this can pass.

Weight progression is exactly linear, so `fontmake`  doesn’t create an `avar`  table.

Sorry for the commit about Haruki Wakamatsu. I PR'd this manually since `gftools packager` is currently not yet able tp PR to the new `main` branch, and this got somehow caught in the act. The information is correct, tho, so feel free to merge it.

### CODE REVIEW

**Comment by davelab6** [comment]
@yanone I'd like to ask you to redo this so the vertical metrics remain the same as before.

**Comment by yanone** [comment]
@davelab6 The UPM has changed with the upgrade, so the numbers are adjusted to the UPM, forgiving rounding errors. I spent quite some time on this. Have you taken this into consideration?

**Comment by yanone** [comment]
@davelab6 I just checked it again visually. I see what you mean. I'm on it.

File: ofl/yrsa/METADATA.pb [hunk 1]
```diff
@@ -1,53 +1,36 @@
 name: "Yrsa"
-designer: "Rosetta"
+designer: "Rosetta, Anna Giedryś, David Březina"
 license: "OFL"
 category: "SERIF"
 date_added: "2016-06-20"
```
Comment (davelab6) [inline ofl/yrsa/METADATA.pb:5]: I reverted this; since it is a minor update, it should not change

File: ofl/yrsa/upstream.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+branch: master
+files:
+  OFL.txt: OFL.txt
+  documentation/Yrsa-DESCRIPTION.en_us.html: DESCRIPTION.en_us.html
+  fonts/YrsaVariable/YrsaVF-Ups.ttf: Yrsa[wght].ttf
+  fonts/YrsaVariable/YrsaVF-Its.ttf: Yrsa-Italic[wght].ttf
+  fonts/Yrsa/Yrsa-Light.ttf: static/Yrsa-Light.ttf
```
Comment (davelab6) [inline ofl/yrsa/upstream.yaml:7]: Was the family hand hinted?

Comment (yanone) [reply ofl/yrsa/upstream.yaml:7]: The current sources are not hand hinted

Comment (davelab6) [reply ofl/yrsa/upstream.yaml:7]: Then no static fonts are needed

Comment (thlinard) [reply ofl/yrsa/upstream.yaml:7]: > Then no static fonts are needed

Considering #2616, are you not putting the cart before the horse? As long as the API is not able to produce usable static fonts, I find that on the contrary having them here is essential.

**Pull request closed without merging**